### ISSUE TITLE
 append_row skips columns

### ISSUE THREAD

**Issue by kahuria00** [burnash/gspread#735]
I am trying to keep adding rows to a google sheet every time data comes in from my web app using append_row() 
but it keeps malfunctioning and skipping columns **

 
![append_row](https://user-images.githubusercontent.com/47794514/77237191-05f05a80-6bd7-11ea-9d37-4afd087a46e3.PNG)

**Expected behavior**

![EXPECTED](https://user-images.githubusercontent.com/47794514/77237227-5f588980-6bd7-11ea-8f7f-1096b86b96b4.PNG)

```
def write_to_dangersheet(flow):
            credentials_file = "file.json"
            scope = ['https://spreadsheets.google.com/feeds',
                     'https://www.googleapis.com/auth/drive']
            credentials = ServiceAccountCredentials.from_json_keyfile_name(credentials_file, scope)
            gc=gspread.authorize(credentials)

            test_wb=gc.open_by_key('yyyyyyyyyyyyyy').worksheet('betaflow')

            f=[phonenumber,date," ",flow,]

            if flow=='webhook-test':
                test_wb.append_row(f)
        write_to_dangersheet(flow)

```

- Operating System : Windows
- Python version: 3.6
- gspread version :3.3.0

**Comment by burnash** [comment]
Thank you for the detailed description.
Looks like it is a similar issue to #537 and was addressed in #726 

The solution is to explicitly specify the table range:

```python
test_wb.append_row(['Test1', '', 'Test2'], table_range='A1')
```

Quick background: this is related to how Google Sheets API treats "virtual" tables. In your example, there are two tables A1:B3 and D1:D3.
More details in [my comment](https://github.com/burnash/gspread/issues/537#issuecomment-586622830) to #537 and in [Google Sheets API guide on appending values](https://developers.google.com/sheets/api/guides/values#appending_values).

Let me know if this helps. I'm closing the ticket for now.

**Comment by kahuria00** [comment]
Thank you for all the help it is working
